_activities/tech-live/002.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Which one looks better as a url ?

`_activities/tech-live/002.md`

vs
`_activities/tech-live/002-hands-on-security-episode-01.md`

Reply:
Second one. It reflects both the title and topic of the page. Yet, I think the leading serial number '002' doesn't look that good.

What we can do are -

1. Override the permalink format stated in 'config.yml' and individually handle permalink in every file. No need to worry about the file name then. Files could be named as '001.md' '002.md' like now to maintain a certain order.

2. Name the file 'hands-on-security-episode-02.md'. This is probably the best way to do it but it would ruin the order if there is any TechLive session on a topic other than "Hands on Security".

Reply:
We have to bring order to the system 😉 

let's at least use the format `_activities/tech-live/002-hands-on-security-episode-01.md`. And then, in a separate commit, we can refactor the overridding of permalinks

Reply:
Closing this pull request then (?) Will create a new pull request with changed names tonight :')

Reply:
I think there is a way to avoid new pull request as long as the branches are same. Otherwise feel free to create a new one.

Reply:
Got it bhai. Pushing into the same branch automatically updates the pull request.